doc/bc/changes-6.9.md

```diff
@@ -3,15 +3,8 @@
 Changes affecting version compatibility with former or future versions.
 
 ## Changes
+- EZP-25679: `ezpublish.api.service.*` now correctly point to the outermost api: SignalSlot
 
+  What changed: All `ezpublish.api.service.*` services are now aliases, not reusing `ezpublish.api.reposiotry` alias to better decouple the repository and correct the services returned.
 
-## Deprecations
```

Comment:
keep the headers, otherwise the structure is lost when copying for next version ;)